sources/VT100Terminal.m

```diff
@@ -2408,6 +2425,7 @@ - (void)setStateFromDictionary:(NSDictionary *)dict {
         return;
     }
     self.termType = dict[kTerminalStateTermTypeKey];
+    self.answerBackString = dict[kTerminalStateAnswerBackStringKey];
```

Comment:
You don't want to pull an NSNull out here. I really should have done this for termType too.
````
if ([self.answerBackString isKindOfClass:[NSNull class]]) {
  self.answerBackString = nil;
}
````